app/src/docs/v1.api-spec.yaml

```diff
@@ -3455,6 +3457,119 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /forms/encryptionKey/algorithms:
+    get:
+      summary: Get the list of encryption algorithms
+      operationId: listEncryptionAlgorithms
+      security:
+        - BasicAuth: []
+        - BearerAuth: []
+          OpenID: []
+      tags:
+        - Form
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    code:
+                      type: string
+                      description: code for encryption Algorithm
+                    display:
+                      type: string
+                      description: display for encryption Algorithm
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /forms/{formId}/encryptionKey/{formEncryptionKeyId}:
+    get:
+      summary: Get the encryption key for a form
+      operationId: readEncryptionKey
+      security:
+        - BasicAuth: []
+        - BearerAuth: []
+          OpenID: []
+      tags:
+        - Form
+      parameters:
+        - in: path
+          name: formId
+          schema:
+            type: string
+            example: bea3b705-1de5-4f4e-a4e6-0716b7674132
+          description: form ID
+          required: true
+        - in: path
+          name: formEncryptionKeyId
+          schema:
+            type: string
+            example: bea3b705-1de5-4f4e-a4e6-0716b7674132
+          description: form encryption key ID
+          required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EncryptionKey'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /forms/{formId}/eventStreamConfig:
+    get:
+      summary: Get a form's event stream configuration
+      operationId: readEventStreamConfig
+      security:
+        - BasicAuth: []
```

Comment:
This should also be removed - the route does not allow API Key (BasicAuth) access.

Reply:
yup got it